linera-service/src/cli_wrappers/remote_net.rs

```diff
@@ -71,6 +70,7 @@ pub struct RemoteNet {
     testing_prng_seed: Option<u64>,
     next_client_id: usize,
     tmp_dir: Arc<TempDir>,
+    close_chain: bool,
```

Comment:
(We could make this type `OnClientDrop` instead of `bool`? Not sure.)

Reply:
Yes, done.